_sidebar.md

```diff
@@ -22,9 +22,9 @@
       - [**Plugin references**](/nodejs-plugin-references.md)
     - Testing Plugins
       - [**Testing Guide**](/testing.md)
-    - JSONRPC
-      - [**JSON RPC Introduction**](/json-rpc.md)
-      - [**JSON RPC Plugin Settings**](/json-rpc-settings.md)
+    - JSON-RPC
+      - [**Introduction**](/json-rpc.md)
+      - [**Plugin Settings**](/json-rpc-settings.md)
```

Comment:
The actual name of the standard is `JSON-RPC`. Repeating the name of the standard in sub-elements is redundant.